jumpstarter/states.py

```diff
@@ -409,7 +454,7 @@ async def _release_resources(event_data: transitions.EventData) -> None:
 async def _maybe_set_event(event_data: EventData, event_name: str) -> None:
     kwargs = _merge_event_data_kwargs(event_data)
     try:
-        event: Event = kwargs[event_name]
+        event: EventType = kwargs[event_name]
```

Comment:
That's definitely not correct. This is an AnyIO event.

Reply:
I renamed the `anyio.Event` at import to not conflict with the `Event` enum. It seemed to me that the having a variable `Event.bootup` and `Event.shutdown` was more useful than the type annotation. By changing the name of the annotation at import, `mypy` will still work, and IDEs will still prompt you to use the correct value. It just avoids the name collision.

Reply:
```
from anyio.abc import Event as EventType
```

Reply:
What is the Event enum used for?

Reply:
I think we should rename that enum to `ActorLifecycleEvent`.
That would be more helpful.